### PR TITLE
LL-4542 - app needs to be updated cta

### PR DESCRIPTION
**Pull request by machard**
https://ledgerhq.atlassian.net/browse/LL-4542

### CODE REVIEW

File: src/components/DeviceAction/rendering.js [hunk 1]
```diff
@@ -280,25 +281,44 @@ export function renderError({
...
-            title={t("common.retry")}
-            onPress={onRetry}
+            title={
+              managerAppName
+                ? t("DeviceAction.button.openManager")
+                : t("common.retry")
+            }
+            onPress={
+              managerAppName
+                ? () => {
```
Comment (valpinkman) [inline src/components/DeviceAction/rendering.js:307]: could you make a useCallback for this function ?

**Review by valpinkman — CHANGES_REQUESTED** [review]

**Review by JunichiSugiura — APPROVED** [review]

**Review by Arnaud97234 — DISMISSED** [review]
When clicking on the CTA, user is redirected to the manager Installed tab on the updates available drawer.
✅

**Review by gre — DISMISSED** [review]
flow errors

**Comment by valpinkman** [comment]
why is there 2000 flow errors again ? @machard

**Review by valpinkman — APPROVED** [review]

**Review by valpinkman — DISMISSED** [review]

**Review by valpinkman — APPROVED** [review]

**Review by Arnaud97234 — APPROVED** [review]

**Pull request merged**